specification/appendix/commitment_discounts.md

```diff
@@ -0,0 +1,278 @@
+
+Spend-based commitment discounts obligate a customer to total monetary quantity over a preselected term, typically denoted by the invoice's selected billing currency (ex: "USD").
+
+## Purchasing
+
+While customers are bound to the term of a commitment discount, cloud-service providers, or CSPs, offer various payment options before and/or during the term:
+
+* *Upfront* - The commitment discount is paid in full before the term begins.
+* *Recurring* - The commitment discount is paid on a repeated basis, typically over each billing period of the term.
+* *Hybrid* - Some of the commitment discount is paid before the term begins, and the rest is paid repeatedly over the term.
+
+For example, if a customer buys a spend-based commitment discount for 1 year, with a $1.00 hourly commitment, and pays with the hybrid option, the commitment discount's payment consists of both:
+
+1. *Upfront* - $4,380 (`24 hours * 365 days * $1.00 * 0.5`)
+2. *Recurring* - $182.50 (`24 hours * 365 days * $1.00 / 12 months`)
+
+## Usage
+
+Commitment discounts follow a "use-it-or-lose-it" model where the amortization of a commitment discount's purchase applies evenly over each charge period of eligible resources over the term.
+
+For example, if a customer buys a spend-based commitment with a $1.00 hourly commitment in January (31 days), only $1.00 is eligible for consumption for each hourly charge period.  This means that if a customer has eligible resources running during this charge period, some or all of the $1.00 that is allocated to the charge period will also be allocated to some or all of these resources.  Conversely, if a customer does not have eligible resources running during this charge period, the $1.00 allocated to the charge period is wasted.
+
+## Commitment Discounts in FOCUS
+
+Within the FOCUS specification, the following examples demonstrate how a commitment discount would appear across various payment and usage scenarios.
+
+### Purchase Rows
+
+All commitment discount purchases appear with a positive `BilledCost`, `PricingCategory` as "Committed", and the commitment discount's id populating both the `ResourceId` and `CommitmentDiscountId` value. Upfront purchases appear as a single record also with `ChargeCategory` as "Purchase", `ChargeFrequency` as "One-Time", and the total quantity and units for commitment discount's term as `CommitmentDiscountPurchasedQuantity` and `CommitmentDiscountUnit`, respectively.
+
+Recurring purchases are allocated across all corresponding charge periods of the term when `ChargeCategory` is "Purchase", `ChargeFrequency` is "One-Time", and `CommitmentDiscountPurchasedQuantity` and `CommitmentDiscountUnit` are reflected only for that charge period.
```

Comment:
Suggestion based on Scenario 2: Recurring

```suggestion
Recurring purchases are allocated across all corresponding charge periods of the term when `ChargeCategory` is "Purchase", `ChargeFrequency` is "Recurring", and `CommitmentDiscountPurchasedQuantity` and `CommitmentDiscountUnit` are reflected only for that charge period.
```

Reply:
Regarding the editorial guidelines, which format are we supposed to use for column IDs — for instance, `ChargeFrequency` or ChargeFrequency? 
Based on https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/501, I assume it's the latter.

Reply:
Will incorporate styling guidelines across all pr's after merging content.